src/main/fc/serial_cli.c

```diff
@@ -617,9 +617,7 @@ const clivalue_t valueTable[] = {
 #if defined(GYRO_USES_SPI) && defined(USE_MPU_DATA_READY_SIGNAL)
     { "gyro_isr_update",            VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP,  &gyroConfig()->gyro_isr_update, .config.lookup = { TABLE_OFF_ON } },
 #endif
-#ifdef GYRO_SUPPORTS_32KHZ
```

Comment:
I think you can still use this define - or something similar, so that it is not available for F1 and F3 :)